options.go

```diff
@@ -28,3 +31,10 @@ func WithLogger(logger LeveledLoggerInterface) Option {
 		c.log = logger
 	}
 }
+
+// WithHTTPClient allows to set a custom http client
```

Comment:
`WithHTTPClient is used to set a custom http client` would sound better. `allows to` isn't the best choice in my opinion